### ISSUE TITLE
Attempted to call method "getName" on class "Twig_Test_Method"

### ISSUE THREAD

**Issue by dnna** [FriendsOfSymfony/FOSCommentBundle#487]
Hello,

I tried including FOSCommentBundle in my project with Symfony 2.6 but it seems to crash Twig with the error mentioned in the title. I'm posting the require part of my composer.json below if it helps:

```
    "require": {
        "php": ">=5.4",
        "symfony/symfony": "2.6.*",
        "cnerta/breadcrumb-bundle": "2.0.*",
        "doctrine/orm": "2.5.*@dev",
        "doctrine/doctrine-bundle": "dev-master",
        "doctrine/doctrine-fixtures-bundle": "dev-master",
        "doctrine/doctrine-cache-bundle": "*",
        "knplabs/knp-paginator-bundle": "2.4.*",
        "liuggio/excelbundle": "2.0.*",
        "twig/extensions": "1.2.*",
        "sunra/php-simple-html-dom-parser": "v1.5.0",
        "symfony/assetic-bundle": "2.6.*",
        "symfony/swiftmailer-bundle": "2.3.*",
        "symfony/monolog-bundle": "2.6.*",
        "sensio/distribution-bundle": "3.0.*",
        "sensio/framework-extra-bundle": "3.0.*",
        "sonata-project/admin-bundle": "2.3.3",
        "sonata-project/doctrine-orm-admin-bundle": "2.3.1",
        "sonata-project/media-bundle": "2.3.1",
        "sonata-project/formatter-bundle": "2.3.3",
        "sonata-project/intl-bundle": "2.2.*",
        "jms/i18n-routing-bundle": "dev-master",
        "jms/security-extra-bundle": "1.5.*",
        "jms/serializer-bundle": "*",
        "friendsofsymfony/user-bundle": "*",
        "friendsofsymfony/rest-bundle": "*",
        "friendsofsymfony/rest": "*",
        "friendsofsymfony/comment-bundle": "2.0.*",
        "nelmio/api-doc-bundle": "2.4.*",
        "stof/doctrine-extensions-bundle": "dev-patch-2",
        "gedmo/doctrine-extensions": "v2.4.0",
        "ruudk/capifony-tools": "dev-master"
    }
```

**Comment by Gadgetdude** [comment]
I noticed the same error message after updating twig/twig from v1.20.0 to v1.21.0. The four \Twig_Test_Method references should be replaced by \Twig_SimpleTest in FOS\CommentBundle\Twig\CommentExtension to solve this problem. This, however, would create a new constraint in composer.json that twig/twig should at least be ~1.12. I'm not sure that this bundle would want to have a new constraint.

Perhaps this problem lies somewhere else. I haven't seen a reference in the twig/twig dependency about this error message. Maybe the problem is caused in combination with another bundle (Assetic?).

**Comment by Gadgetdude** [comment]
After some more tests I created a new issue report for twig/twig here. A temporary solution is to downgrade the twig/twig version to "~1.20.0" in your project.

Issue report: https://github.com/twigphp/Twig/issues/1791

**Comment by Gadgetdude** [comment]
The project owner of Twig confirmed that it is a regression and they want to provide a fix a soon as possible.

@XWB: This issue can be closed as it is an issue with twig/twig v1.21.0.

**Comment by XWB** [comment]
:+1:
